api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -431,6 +432,15 @@
      */
     ClientBuilder setRetryMaxAttempts(int maxAttempts);
 
+    /**
+     * Sets the {@link RequestExecutorFactory},
+     * otherwise it will be loaded as a Service / SPI via the {@link RequestExecutorFactory} class
+     *
+     * @param requestExecutorFactory that should be used to create the RequestExecutor
+     * @return the ClientBuilder instance for method chaining
+     */
```

Comment:
nit: correct the javadoc syntax (end with a period).

```suggestion
    /**
     * Sets the {@link RequestExecutorFactory}, otherwise it will be loaded as a Service / SPI 
     * via the {@link RequestExecutorFactory} class.
     *
     * @param requestExecutorFactory that should be used to create the RequestExecutor
     * @return the ClientBuilder instance for method chaining
     */
```